templates/content/basic/src/MyProvider.Runtime/MyProvider.Runtime.fsproj

```diff
@@ -13,10 +15,22 @@
     <None Include="paket.references" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.5.2" />
     <ProjectReference Include="..\MyProvider.DesignTime\MyProvider.DesignTime.fsproj">
       <IsFSharpDesignTimeProvider>true</IsFSharpDesignTimeProvider>
       <PrivateAssets>all</PrivateAssets>
     </ProjectReference>
   </ItemGroup>
   <Import Project="..\..\.paket\Paket.Restore.targets" />
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <Target Name="BeforeBuild">
```

Comment:
Same here, these are not necessary @dsyme

Reply:
Why?  The design time component is not places in the right location AFAICS.  WHat loads it?

Reply:
(I agree the BeforeBuild is not necessary, but not the AfterBuild)

Reply:
It is `FSharpToolsDirectory>typeproviders</FSharpToolsDirectory>` and `<PackagePath>typeproviders</PackagePath>`, you can verify that files are in the correct place. At least that is what I did here when adding the change.

It was a while ago so please correct me if I'm wrong and you don't see that on your local build. But I really really want to remove these msbuild extesibility stuff if it's possible

Reply:
Documented in https://github.com/fsprojects/FSharp.TypeProviders.SDK/pull/339#issuecomment-716881125